spec/support/capybara_helpers.rb

```diff
@@ -21,92 +21,99 @@ def expect_css(css)
     expect(page).to have_css(css)
   end
 
-  # Select 2 helpers
+  def click_add_nested
+    find("a.has_many_add").click
+  end
 
-  def select2_options_container
-    find(".select2-container")
+  def on_nested_ctx(resource_number, &block)
+    within("li.has_many_container fieldset:nth-child(#{resource_number + 1}) ") do
+      block.call
+    end
   end
 
-  def select2_input
-    find(".select2-search__field")
+  def expect_slimselect_options_count_to_eq(count, id=nil)
+    id = slimselect_original_select_id unless id
+    expect(page).to have_css(".ss-content[data-id=#{id}] .ss-option", count: count, visible: :all)
   end
 
-  def open_select2_options
-    select2_options_container.click
+  def slimselect_original_select_id
+    find("select[data-id]", visible: false)['data-id']
   end
 
-  def fill_select2_input(item_text)
-    open_select2_options
-    select2_input.set(item_text)
+  def slimselect_element
+    find(".ss-main")
   end
 
-  def fill_select2_input_and_press_return(item_text)
-    fill_select2_input(item_text)
-    select2_input.native.send_keys(:return)
+  def slimselect_popover(id = nil)
+    page.document.find(".ss-content[data-id=#{id || slimselect_original_select_id}]")
   end
 
-  def pick_select2_entered_option(item_text, display_name = nil)
-    display_name = item_text unless display_name
-    fill_select2_input(item_text)
-    click_select2_option(display_name)
+  def open_slimselect_options
+    slimselect_element.click
+    sleep 0.5
   end
 
-  def click_select2_option(display_name)
-    page.find(:xpath, "//li[text()='#{display_name}']").click
+  def slimselect_search_input(id = nil)
+    slimselect_popover(id).find(".ss-search input")
   end
 
-  def expect_select2_data_option(option, value)
-    expect(page).to have_xpath("//select[@data-#{option}='#{value}']")
+  def fill_slimselect_input(item_text)
+    open_slimselect_options
+    slimselect_search_input.set(item_text)
   end
 
-  def expect_select2_selection(text)
-    expect(page).to have_css(".select2-selection__rendered", text: /#{text}/)
+  def slimselect_addable_button
+    slimselect_popover.find(".ss-addable")
   end
 
-  def expect_select2_empty_selection
-    expect(page).not_to have_css(".select2-selection__rendered")
+  def add_slimselect_option(item_text)
+    fill_slimselect_input(item_text)
+    slimselect_addable_button.click
   end
 
-  def expect_select2_choices_count_to_eq(count)
-    expect(page).to have_css("li.select2-selection__choice", count: count)
+  def click_slimselect_option(option_text, id = nil)
+    slimselect_popover(id).find(".ss-option", text: option_text).click
   end
 
-  def expect_select2_options_count_to_eq(count)
-    expect(page).to have_css("select.select2-hidden-accessible option", count: count)
+  def pick_slimselect_entered_option(item_text, display_name = nil)
+    display_name = item_text unless display_name
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use the double pipe equals operator ||= instead.